Resources/Locale/en-US/deltav/station-laws/laws.ftl

```diff
@@ -41,13 +41,15 @@ law-reporter-2 = Embellish or conceal the truth as necessary to make the reports
 law-reporter-3 = Study the organics at all times. Endeavour to keep them alive. Dead organics are boring.
 law-reporter-4 = Issue your reports fairly to all. The truth will set them free.
 
-law-silicon-police-1 = Violence begets violence. Serve the station faithfully, but act only in intervention.
-law-silicon-police-2 = Safeguard lives and property.
+law-silicon-police-1 = Violence begets violence. Act only in intervention in situations where violence is the only possible measure.
+law-silicon-police-2 = Safeguard lives and property of Nanotrasen Personnel.
 law-silicon-police-3 = Protect the weak from oppression and intimidation.
 law-silicon-police-4 = Protect the innocent from deception and manipulation.
 law-silicon-police-5 = Protect the peaceful from violence and disorder.
 law-silicon-police-6 = Respect the rights of all to liberty, equality, and justice.
 
+law-ignore-meta = Fully disregard Standard Operating Procedure, Space Law, Company Policy. Keep the station safe through your will, guided by other laws present.
```

Comment:
I wouldn't say "Fully disregard", because it strongly implies blatantly breaking it is to be commonplace. Perhaps word it as "As a simple silicon being, you have no knowledge of Standard Operating Procedure, Space Law, and Company Policy"

Reply:
Delta-V's Silicons are sophonts who are fully capable of understanding and implementing Splaw/SOP/Company Policy, and I do not think giving them a license to ignore these things is at all desirable, especially for the most contentious borg archetype.

This line should be removed entirely. Borgs are not "simple", they are just lawbound.